### DISCUSSION TITLE
macOS: I believe, ImageMagic distribution only runnable on 10.15 High Sierra is actually an issue

### DISCUSSION THREAD

**Discussion by singalen** [ImageMagick/ImageMagick#1887] [General]
Let me raise [this topic from the forum](https://imagemagick.org/discourse-server/viewtopic.php?f=1&t=37278&p=173676#p173676) again.

A lot of users stay away from high Sierra because it drops the 32-bit application support. Distribution for 10.15-only is useless for a lot of people.

I prefer not to use Homebrew or MacPorts installation, because it installs a lot of X11 libraries, which I'm trying to avoid on my system. A downloadable package is easy to uninstall, unlike Homebrew/Macports dependencies.

Compilation targeting the lower versions of Mac SDK is easy: https://stackoverflow.com/questions/25352389/what-is-the-difference-between-macosx-deployment-target-and-mmacosx-version-min

I was able to compile it on 10.14 for 10.6:

```
$ CFLAGS=-mmacosx-version-min=10.6 LDFLAGS=-mmacosx-version-min=10.6 ./configure
...
$ make -j16 && sudo make install
...

$ otool -l $(which convert)
...
Load command 9
      cmd LC_VERSION_MIN_MACOSX
  cmdsize 16
  version 10.6
      sdk 10.14
...
```

Should I create an issue?

**Comment by urban-warrior** [comment]
We're good.  We'll update the release this weekend based on your flags.   FYI, our build tool chain requires a minimum of 10.9.
